src/Illuminate/Foundation/Testing/TestResponse.php

```diff
@@ -450,12 +450,13 @@ protected function assertJsonMessage(array $data)
      * Assert that the response has the exact given JSON.
      *
      * @param  array  $data
+     * @param  bool   $preserveEmptyObjects
      * @return $this
      */
-    public function assertExactJson(array $data)
+    public function assertExactJson(array $data, bool $preserveEmptyObjects = false)
```

Comment:
I don't think there is a need to pass in a separate boolean parameter.

Reply:
What would you recommend instead?